scripts/common/install_ccache.sh

```diff
@@ -29,7 +29,7 @@ ln -s /usr/local/bin/ccache /usr/local/bin/_ccache/clang
 # Create activate_ccache.sh script
 cat <<EOF > /usr/local/bin/activate_ccache.sh
 #!/usr/bin/env bash
-export PATH=/usr/local/bin/_ccache:$PATH
+PATH=/usr/local/bin/_ccache:$PATH
```

Comment:
I think that change could cause issues for cmake subprocesses that will need ccache to be found...

Reply:
Whoops, I think I messed this up. The intended change was:

```
PATH=/usr/local/bin/_ccache:$PATH
export PATH
```

because of https://github.com/koalaman/shellcheck/wiki/SC2155 - however, there is no subshell `$()` here. I think I realized this halfway and forgot to revert the removal of export. Should be undone:

```suggestion
export PATH=/usr/local/bin/_ccache:$PATH
```